Sources/PostgresNIO/New/PostgresQuery.swift

```diff
@@ -67,6 +67,11 @@ extension PostgresQuery {
             try self.binds.append(value, context: context)
             self.sql.append(contentsOf: "$\(self.binds.count)")
         }
+
+        @inlinable
+        public mutating func appendInterpolation(unsafeSQLInjection interpolated: String) {
```

Comment:
I think the word `Injection` is overkill in this case - `unsafe` is at least theoretically supposed to be a sufficiently big signal flare in _any_ Swift code to cover it. I'd tend towards making the label `unsafeSQL` and then clearly defining SQL injection and its dangers (in short form, not one of my rants!! 😆) as a documentation comment.

Reply:
However unsafeSQL doesn't give users such clear google results. SQL injection results are very clear...

Reply:
Yeah I agree `unsafeSQLInjection` is confusing. We can add it as a comment if we're concerned about searching. I like `\(unsafeRaw: "")` or `\(raw: "")` - lines up with the rest of the cases across Vapor where we allow unsafe injection of variables and still makes it clear

Reply:
I think the full name is too wordy. Surely we can detail the issues with sql injection in the comments. I would go further than @gwynne i don't think SQL is required. It should be obvious it is SQL as it is inside a SQL query. So `\(unsafe: ...)` or `\(unsafeRaw...)` should be enough.

Reply:
I’ll put in a vote for ‘unsafeRaw’ in that it suggests direct SQL phrases without escaping or other processing for safety.

Reply:
I talked to @weissi about this yesterday, and we came up with `unescaped`, which I like the most so far. @0xTim, @gwynne, @adam-fowler, @Jerry-Carter What do you think about this?

Reply:
Yeah I'm happy with that

Reply:
Sounds good

Reply:
I want to argue for `unsafeUnescaped`, but not enough to die on the hill; I can live with this 🙂